homeassistant/components/reolink/services.yaml

```diff
@@ -16,3 +16,30 @@ ptz_move:
           min: 1
           max: 64
           step: 1
+
+play_chime:
+  fields:
+    device_id:
+      required: true
+      selector:
+        device:
+          multiple: true
+          filter:
+            integration: reolink
+            model: "Reolink Chime"
+    ringtone:
+      required: true
+      selector:
+        select:
+          translation_key: ringtone
+          options:
+            - citybird
+            - originaltune
+            - pianokey
+            - loop
```

Comment:
Maybe sort these 🔡?

Reply:
I kept the same order as in the Reolink app.
Translations will screw up the alphabetic order anyway (I think).

Reply:
I wasn't worried about sorting them for the users but for us developers when reading this file.